json-endpoints/jetpack/class.jetpack-json-api-plugins-install-endpoint.php

```diff
@@ -38,6 +38,12 @@ protected function install() {
 		}
 
 		if ( ! $this->bulk && isset( $error ) ) {
+
+			if ( $error_code === 'download_failed' && $error === 'Download failed.' ) {
```

Comment:
@roccotripaldi isn't `Download failed.` the translatable string in `class-wp-upgrader.php`? if user translates it this condition will fail.